tests/Mockery/ExpectationTest.php

```diff
@@ -402,6 +402,31 @@ public function testNeverCalled()
         $this->mock->shouldReceive('foo')->never();
     }
 
+    public function testShouldNotReceive()
+    {
+        $this->mock->shouldNotReceive('foo');
```

Comment:
Should you be calling `$this->container->mockery_verify();` here as well and verify, that if more than 0 calls are made we're getting an exception?